package NQueryDataSaaS;

import "mapreduce/yt/interface/protos/extension.proto";

message TQDSaaSInputMeta {
    // Пространство имён данных
    optional string Namespace = 1  [(NYT.key_column_name) = "Namespace"]; // NQueryData::TSourceFactors.SourceName (бывший "-S" в querydata_indexer)

    // Таймстемп данных в микросекундах
    optional uint64 Timestamp_Microseconds = 2  [(NYT.key_column_name) = "Timestamp_Microseconds"]; // NQueryData::TSourceFactors.Version
}

message TQDSaaSInputMetas {
    repeated TQDSaaSInputMeta Metas = 1;
}
